src/main/java/duke/DeleteTask.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+/**
+ * Delete Task class with int index that represents which task in TaskList to delete
+ *
+ * @author wj331
+ */
+public class DeleteTask extends Command {
+    private int indexToDelete;
```

Comment:
```suggestion
    private int index;
```
Perhaps naming it as index could be more concise and still clear.